code/modules/mob/living/simple_animal/hostile/mimic.dm

```diff
@@ -181,7 +190,15 @@ var/global/list/protected_objects = list(/obj/structure/table, /obj/structure/ca
 		icon_living = icon_state
 		overlays = O.overlays
 
-		if(istype(O, /obj/structure) || istype(O, /obj/machinery))
+		if(istype(O, /obj/structure))
+			health = (anchored * 50) + 50
+			destroy_objects = 1
+			if(O.density && O.anchored)
+				knockdown_people = 1
+				melee_damage_lower *= 2
+				melee_damage_upper *= 2
+		else if(istype(O, /obj/machinery))
+			is_electronic = 1

```

Comment:
Wouldn't it be easier to just have a second istype(O, /obj/machinery) and have is_electronic set to 1 for that, so there's less duplicate code?